fineract-provider/src/main/java/org/apache/fineract/notification/service/NotificationReadPlatformServiceImpl.java

```diff
@@ -107,6 +107,13 @@ public void updateNotificationReadStatus() {
         this.jdbcTemplate.update(sql, appUserId);
     }
 
+    @Override
+    public void updateSingleNotificationReadStatus(Long id) {
+        String sql = "UPDATE notification_mapper SET is_read = true WHERE is_read = false AND id = ?";
```

Comment:
I know the existing method uses native query but please don't introduce it in one more method.
Please rely on JPA for updating the entity. Thanks.